UITests/BookmarksPanelTests.swift

```diff
@@ -7,7 +7,6 @@
 import Shared
 import Foundation
 import WebKit
-import EarlGrey
 /*
 class BookmarksPanelTests: KIFTestCase {
```

Comment:
This test suite is disabled since we changed to use Bookmark Rust component and it started to fail. I have just removed the EarlGrey import, not the methods in the tests in case we re-visit this to either re-enable using KIF or use XCUITests. I will open an issue to follow this up.